visualizations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -182,12 +182,16 @@ class FanChart(FilteredPlotly):
         Expects `index` parameter to be used as 'x' value, a `name` parameter
         to correspond with a name in the data dataframe, a `value` and `value`
         that will determine the size of the marker (in height)
-    :param kwargs: optional `xaxis` and `yaxis` paramameter. Will create a
-        label for the given axis. Defaults to `None`.
+    :param xaxis: Will create a label for the x-axis. Defaults to `None`.
+    :param yaxis: Will create a label for the y-axis. Defaults to `None`.
+    :param logx: boolean value to toggle x-axis logarithmic scale
+    :param logy: boolean value to toggle y-axis logarithmic scale
```

Comment:
Add information regarding default value? E.g.

`:param logx: If to use logarithmic x-axis or not. Defaults to `False`.`

Reply:
Done 👍